GenderPayGap.WebJob/Functions/FrequentTimerTriggeredWebJobs/Functions.SendReminderEmails.cs

```diff
@@ -150,8 +152,8 @@ private static void SaveReminderEmailRecord(User user, SectorTypes sectorType, b
         {
             var reminderEmailRecord = new ReminderEmail
             {
-                UserId = user.UserId,
-                SectorType = sectorType,
+                UserId = user.UserId,
+                SectorType = sectorType,
```

Comment:
I'm very confused as to why this is here. It's not showing in gitkraken

Reply:
There was a trailing space on the end of these two lines.
You can toggle this button to show/hide whitespace changes in GitKraken
![image](https://user-images.githubusercontent.com/7769588/76631708-05522700-653a-11ea-9f4c-c85521fcb09e.png)

Reply:
Neat thanks